src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/AuthServiceConfig.java

```diff
@@ -39,14 +39,17 @@ public AuthTokenValidator tokenValidator(ServiceAuthorisationApi s2sApi) {
     @ConditionalOnProperty(name = "idam.s2s-auth.url", havingValue = "false")
     public AuthTokenValidator tokenValidatorStub() {
         return new AuthTokenValidator() {
+            @Override
             public void validate(String token) {
                 throw new NotImplementedException();
             }
 
+            @Override
             public void validate(String token, List<String> roles) {
                 throw new NotImplementedException();
             }
 
+            @Override
             public String getServiceName(String token) {
                 return "some_service_name";
```

Comment:
Why are we overriding the above methods?

Reply:
methods were always overridden, just adding annotation to make it clear (as suggested by PMD)

Reply:
oh, got it. didn't notice the interface earlier